src/libs/examples/Holmakefile.gen

```diff
@@ -0,0 +1,28 @@
+# includes
+# ----------------------------------
+INCLUDES   = ../../core ../../theories ../
```

Comment:
you can save these includes. it makes the tests more stable in this way because you include the heap anyways and you will notice when something is missing in the heap. since you have a heap, nothing that is included will be loaded from the other directories anyways if i understand it correctly.

another way would be to take another heap down there, for example the theories heap and only include the parent directory here. but i'm not sure whether this is a big improvement considering all.

let me know what you think